mkdocs_git_revision_date_localized_plugin/util.py

```diff
@@ -224,3 +239,19 @@ def add_spans(date_formats: Dict[str, str]) -> Dict[str, str]:
                 % (date_type, date_string)
             )
         return date_formats
+
+    @staticmethod
+    def parse_git_ignore_revs(filename: str) -> List[str]:
+        """
+        Parses a file that is the same format as git's blame.ignoreRevsFile and return the list of commit hashes.
+
+        Whitespace, blanklines and comments starting with # are all ignored.
+        """
+        result = []
+        with open(filename, "rt") as f:
```

Comment:
perhaps we should specify UTF-8 encoding. This is standard for mkdocs. See f.e. https://github.com/mkdocs/mkdocs/blob/df3739d51903ab56771ac071a05b5aa9cdf9e129/mkdocs/commands/new.py#L43

Reply:
Also, what if this file is not found, because a user specified the wrong path? Can we give a helpful error, showing with path was tried and which parameter in mkdocs.yml should be changed?